templates/SpecialBrowse.mustache

```diff
@@ -0,0 +1,40 @@
+{{!
+  Special:Browse template
+
+  Required data:
+  - subject: string - encoded JSON for ajax loading
+  - options: string - encoded JSON for ajax loading
+  - html-output: string - Raw HTML that does not belong to any elements below
+  - html-noscript: string - HTML of the noscript message box
+  - data-factbox: object - Factbox data (see Factbox.mustache)
+  - data-form: object - SMW browse search form
+    - form-action: string
+	- input-value: string
+	- input-placeholder: string
+	- button-value: string
+	- form-title: string
+}}
+<div
+	class="smw-browse"
+	{{#subject}}data-mw-smw-browse-subject="{{.}}"{{/subject}}
+	{{#options}}data-mw-smw-browse-options="{{.}}"{{/options}}
+>
+	{{#html-output}}{{{.}}}{{/html-output}}
+	{{#html-noscript}}<noscript>{{{.}}}</noscript>{{/html-noscript}}
+	<div class="smw-browse-content">
+		{{#data-factbox}}{{>Factbox}}{{/data-factbox}}
+		{{#data-form}}
+		<form class="smw-browse-search" name="smwbrowse" action="{{{form-action}}}" method="get" role="search">
```

Comment:
Why are the attributes variables here and below not escaped?

Reply:
Done